src/Symfony/Component/HttpFoundation/ParameterBag.php

```diff
@@ -99,54 +99,20 @@ public function add(array $parameters = array())
      * @throws \InvalidArgumentException
      *
      * @api
+     *
+     * @deprecated Finding deep items is deprecated since version 2.7, to be removed in 3.0.
```

Comment:
Since the method itself is not deprecated we should avoid using `@deprecated` tag here.